WireDomain/Sources/WireDomain/Repositories/UpdateEvents/UpdateEventsLocalStore.swift

```diff
@@ -66,6 +66,14 @@ final class UpdateEventsLocalStore: UpdateEventsLocalStoreProtocol {
         )
     }
 
+    public func storeServerTimeDelta(
+        _ serverTimeDelta: TimeInterval
+    ) async {
+        await syncContext.perform { [syncContext] in
+            syncContext.serverTimeDelta = serverTimeDelta
```

Comment:
We shouldn't be storing this on the context, I know it's already done this way so no blocker of course.

Reply:
I'll revisit this with async notifications @johnxnguyen